source/guides/getting-started.md

```diff
@@ -713,15 +713,16 @@ require 'spec_helper'
 require_relative '../../../../apps/web/controllers/books/create'
 
 describe Web::Controllers::Books::Create do
+  let(:action) { Web::Controllers::Books::Create.new }
   let(:params) { Hash[book: { title: 'Confident Ruby', author: 'Avdi Grimm' }] }
 
-  after do
+  before do
     BookRepository.clear
   end
 
   it 'creates a new book' do
     action.call(params)
-    action.book.id.wont_be_nil
+    BookRepository.all.count.wont_equal 0
```

Comment:
@Bounga I don't think this is a good idea, because it references `BookRepository` directly.

If you change data store, or want to use a stub repository to inject for this test, you'll be in trouble. It also requires that your database table **must** be empty before the test, otherwise you'll have a false positive pass.

Exposures are a great way to unit test actions, their usage should be encouraged.

Can you please remove this change, and I'll merge it. Thank you! :smile: